ipatests/test_integration/test_installation.py

```diff
@@ -670,6 +670,13 @@ def get_ipa_services_pids(host):
 ##
 # Rest of master installation tests
 ##
+def master_unistalled(function):
```

Comment:
As this decorator is actually a fixture, what do you think about defining it as a pytest.fixture?
You also should wrap `function()` in try-finally block (not needed if using pytest.fixture)

Reply:
Good point, let me rewrite it as a fixture

Reply:
1. Fixtures do have "scope" parameter
2. Fixtures are applied to each test case individually unless the `autouese` parameter is `True` (False by default)
3. You do need to uninstall ipa server in every test case you have added